xarray/core/combine.py

```diff
@@ -635,16 +635,14 @@ def _combine_single_variable_hypercube(
     return concatenated
 
 
-# TODO remove empty list default param after version 0.19, see PR4696
```

Comment:
same as with the `lock` kwarg: I read this as "this needs to be done in the version after 0.19".

Reply:
actually, since the deprecation was introduced in #4696, which has been merged about three weeks ago and thus has not been included in any release, yet, I think we should update the scheduled version to 0.21 and keep the deprecation?

Reply:
Good point - I just saw 0.19 and removed it. Do you want to roll that one back and change 0.19->0.21?